tables/themes.md

```diff
@@ -1 +1,28 @@
-placeholder
+
+# tables/themes
+
+Note: Currently the `themes` table is a WIP so this information will need to be updated
+
+The `themes` table exists to help categorise topics into different parent themes that are represented with an id. This will help end users find common topics that they are interested in querying. It is still a WIP but eventually there will be something like `theme_id` in the `topics` table that will relate the two in a one to many relationship.
+
```

Comment:
Would be good to have an example here similar to the one in top_level_geographies? (under a header of 'Example use') Sometimes aids understanding for people when it's placed in context.

Reply:
think the problem atm is that it needs a foreign key setup for it to be useful at this point? as without a reference in topics it's hard to do an example involving other tables

Reply:
Ah that's fair - maybe worth putting the header in there with TBC or similar? Then we won't forget down the line!

Reply:
yea good idea, I'll add it in then

Reply:
done